content/blog/2023/when-to-choose-a-progressive-web-app.md

```diff
@@ -0,0 +1,188 @@
+---
+title: When to Choose a Progressive Web App
+sub: Part 3 -- Responsive Web App vs PWA vs Native Mobile App
+author: sondra
+date: 2023-07-05
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part3-progressive.jpg
+  alt: Pen and notebook with a sketch of a website interface
+summary: |
+  If you're weighing the performance optimization
+  and device integration opportunities
+  of a native mobile app
+  against the broad reach and lower cost
+  of a responsive web app,
+  and can't decide which is a higher priority
+  for your digital product,
+  don't despair.
+  A progressive web app may be
+  just the solution you need.
+  This is part 3 of a three-part series
+  unpacking the reasons to choose
+  one platform over another.
+  Let's explore the progressive web app.
+---
+{% import 'embed.macros.njk' as embed %}
+
+If you haven't read parts 1 and 2 yet, you may want to start with the
+introduction to RWAs, PWAs, and Native Mobile Apps in part 1. There's also a
+comparison chart for quick reference.
+
+**Read all three parts to compare app types:**
+
+1. [When to Choose a Responsive Web App](/2023/05/05/when-to-choose-a-responsive-web-app/)
+2. [When to Choose a Native Mobile App](/2023/06/07/when-to-choose-a-native-mobile-app/)
+3. [When to Choose a Progressive Web App](/2023/07/05/when-to-choose-a-progressive-web-app/)
+
+## Progressive Web App -- Best of Both Worlds
+
+> **TL;DR**
+>
+> If your top priority is some combination of wide reach and
+> advanced capabilities, you will want to investigate the
+> progressive web app (PWA) option for your project.
+
+A progressive web app (PWA) is a responsive web app with additional capabilities
+similar to a native mobile app. Like responsive web apps, PWAs are built using
+standard web technologies -- HTML, CSS, and JavaScript. They are accessible from
+any device with a web browser such as Chrome, Firefox, Microsoft Edge, Safari,
+or a mobile browser. Like native mobile apps, PWAs can be included in app
+stores. They can provide push notifications, offline mode, and a home screen
+icon.
+
+If you decide to build a PWA, you will be in the company of many of the biggest
+companies. Google, AirBnB, Twitter, and Microsoft use PWAs for their products
+alongside or in place of native apps. Check out this [directory of
+PWAs](https://www.findpwa.com/list/top-apps). You may discover you are already
+using one!
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/yes.png',
+    alt: 'The word "yes" with a smiley and sparkles'
+  }],
+  attrs={
+    'style': '--column: content-start'
+  }
+) }}
+
+## Choose a PWA when…
+
+### Lower Development & Maintenance Costs
+*…when it's important to keep initial development costs low.*
+
+Just like responsive web apps, progressive web apps require just one
+[codebase](/2023/05/05/when-to-choose-a-responsive-web-app/#choose-an-rwa-when%E2%80%A6)
```

Comment:
I'm not sure this link is adding much -- we're basically re-stating the same info here.